docker/ci/setup.sh

```diff
@@ -131,13 +131,10 @@ EOF
                 yum -q -y install \
```

Comment:
This file looks good,but can you move everything else into a separate pull request?

Reply:
I split off the changes to this file to https://github.com/timescale/timescaledb-toolkit/pull/670.